save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -442,6 +443,23 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
         }
     }
 
+    private fun getUrlWithFiltersParams(filterValue: TestExecutionFilters): String {
+        val hrefFirst: String = window.location.href.split('?')
+            .first()
+        val filtersList = listOf("status=" to filterValue.status?.name, "fileName=" to filterValue.fileName, "testSuite=" to filterValue.testSuite, "tag=" to filterValue.tag)
+            .filter {
+                it.second?.isNotBlank() ?: false
+            }
```

Comment:
May be it will more clear
```suggestion
        val filtersList = listOf(
            "status=" to filterValue.status?.name,
            "fileName=" to filterValue.fileName,
            "testSuite=" to filterValue.testSuite,
            "tag=" to filterValue.tag
        )
            .filter {
                !it.second.isNullOrBlank()
            }
```

Reply:
```kotlin
val filtersList = listOfNotNull(filterValue.status?.name?.let { "status=${filterValue.status?.name}" },
            filterValue.fileName?.let { "fileName=${filterValue.fileName}" },
            filterValue.testSuite?.let { "testSuite=${filterValue.testSuite}" },
            filterValue.tag?.let { "tag=${filterValue.tag}" })
```
I did this